### PR TITLE
complete frogger-game

### PR DESCRIPTION
**Pull request by drewzag**
### Frogger Game

[Demo](https://drewzag.github.io/frontend-nanodegree-arcade-game/) |
[Code base](https://github.com/drewzag/frontend-nanodegree-arcade-game)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by OleksiyRudenko** [comment]
See what bot asked to do. It is unlikely that the code will be reviewed until you accomplish self-check and fixes and explicitly confirm that you did this.

**Comment by drewzag** [comment]
My bad, I missed bot's recommendations.

Self-check done!

And I have question about this
> Const objects help organizing and structure const data even better, e.g. const PLAYER_CONF = { initialPosition: {x: 1, y: 5}, sprite: '...', ...etc... };

Why is it better? In this case I can accidentally redefine elements in object and don't even get an error

**Comment by OleksiyRudenko** [comment]
> > Const objects help organizing and structure const data even better, e.g. const PLAYER_CONF = { initialPosition: {x: 1, y: 5}, sprite: '...', ...etc... };
> 
> Why is it better? In this case I can accidentally redefine elements in object and don't even get an error

It is not better in general. It says "Const objects help **organizing and structure const data even better**"
Think of big configurations that you have in a dedicated file. Now you want to import config from that file.
Your alternatives can be
```
// 1
import { GAME_SETTINGS } from `./config.js'

// 2
import { FIELD_SETTINGS, PLAYER_SETTINGS, ENEMY_SETTINGS } from `./config.js'

// 3
import {
FIELD_ROW = 5,
FIELD_COLS = 5,
CELL_X_SIZE = 101,
....
ENEMY_MIN_SPEED,
ENEMY_MAX_SPEED,
....
PLAYER_START_COL,
PLAYER_START_ROW,
PLAYER_SIZE_Y,
...
} from `./config.js'
```

So just pick what works better for the project, readability and code maintenance.

File: submissions/drewzag/frogger-game/app.js [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+Player.prototype.render = function () {
+  ctx.drawImage(Resources.get(this.sprite), this.x, this.y)
+}
+
+const player = new Player(PLAYER_START_X, PLAYER_START_Y, PLAYER_SPRITE)
+const allEnemies = []
+for (let i = 0; i < 3; i++) {
+  let count = i
+  if (i >= 3) {
```
Comment (OleksiyRudenko) [inline submissions/drewzag/frogger-game/app.js:142]: Does this ever happen?
Minor issue.

**Review by OleksiyRudenko — APPROVED** [review]
@drewzag perfect job done!

File: submissions/drewzag/frogger-game/app.js (hunk 1 above)
Comment (drewzag) [reply submissions/drewzag/frogger-game/app.js:142]: I've played more than 3 enemies. I'll fix this! 
Thanks for your review)

**Pull request merged**